torch_xla/distributed/xla_backend.py

```diff
@@ -70,7 +70,7 @@ def allreduce(self, tensors, all_reduce_options):
     xm.all_reduce(reduce_type, tensors, groups=self._mesh, pin_layout=False)
     return _ret_work(tensors)
 
-  def allgather(self, output_tensors_list, input_tensors):
+  def allgather(self, output_tensors_list, input_tensors, opts=None):
```

Comment:
@will-cromar I think we need to add some comment around `opts=None`. We might think that this unused argument needs to be remove in the future -- some linter would complain, too.

Reply:
Is a comment here necessary? This is consistent with the Python interface it is implementing here: https://github.com/pytorch/pytorch/blob/a85d1f0bcdd02cf18d3b0517337458cb51a18cdb/torch/_C/_distributed_c10d.pyi#L212-L217

Do you think we should like to this `pyi` file somewhere in the code?

Reply:
I see, if we are implementing the abstract class, then we don't need to -- had to view the entire file to see `class ProcessGroupXla(ProcessGroup)` lol. Thanks @will-cromar